GameData/RealismOverhaul/RO_SuggestedMods/StationPartsExpansion/RO_SSPX_338m_Parts.cfg

```diff
@@ -65,8 +65,7 @@
 	
 	@MODULE[ModuleProbeControlPoint] {}
 	@MODULE[ModuleCommand] {}
-	@MODULE[ModuleDataTransmitter] {}
```

Comment:
These rows all look weird to begin with, not sure if this is the intended behaviour

Reply:
I don't know what this is trying/intending to do, either.  Should those be `%MODULE` to guarantee they exist?  Or are they just NOPs to be deleted?
Why are we NOPing a ModuleCommand and then patching an MDT to be a ModuleCommand?
Are there supposed to be *2* ModuleFuelTanks on here?  Or just 1?  (This has a weird pattern of creating a node and then immediately patching it).
@pap1723 ?

Reply:
Maybe the intention was to delete `ModuleCommand` (rather than edit) and then create a new one (i.e., add `MODULE` before the `{`)?

Reply:
Semi-confirmed, that's close to what the `225m` part does:

https://github.com/KSP-RO/RealismOverhaul/blob/743d917814d5bbbad015404a2bcc013776ad1f6d/GameData/RealismOverhaul/RO_SuggestedMods/StationPartsExpansion/RO_SSPX_225m_Parts.cfg#L53-L61

Reply:
Thanks.  TODO for @DRVeyl : Fix the SSPX configs with a more appropriate pattern, they're making this mistake everywhere in addition to only a couple of the files having invalid syntax.